code/src/java/pcgen/gui2/converter/panel/RunConvertPanel.java

```diff
@@ -131,93 +131,88 @@ public boolean performAnalysis(final CDOMObject pc)
 		}
 		sortCampaignsByRank(totalCampaigns);
 
-		new Thread(new Runnable()
-		{
-			@Override
-			public void run()
+		new Thread(() -> {
```

Comment:
IMHO taking a huge anonymous thing and making another huge anonymous thing is kind of meh.  Something that is almost 100 lines long probably belongs in a named class? Thoughts?